pkg/controllers/dynakube/activegate/internal/statefulset/builder/modifiers/rawimage_test.go

```diff
@@ -40,7 +41,7 @@ func TestRawImageModify(t *testing.T) {
 		dynakube := getBaseDynakube()
 		enableKubeMonCapability(&dynakube)
 		setRawImageUsage(&dynakube, true)
-		mod := NewRawImageModifier(dynakube)
+		mod := NewRawImageModifier(dynakube, prioritymap.NewMap())
```

Comment:
```suggestion
		mod := NewRawImageModifier(dynakube, prioritymap.NewMap())
```
btw you can just use `prioritymap.New()` because it's already in package name.